rfcs/text/0428-extending-types.md

```diff
@@ -111,31 +120,64 @@ occupation◄───────────────┘
+In the proposed [Versioning RFC](./0408-versioning-types.md) for the type system, having `{ "additionalProperties": false }` for all schemas is an assumption made for determining type compatibility, which means that any supertype will not validate against a subtype that adds properties if it receives all properties of a subtype instance. For example, if we supply the `Employee` instance from above to a `Person`, it will receive properties that are considered `additionalProperties` (the `occupation` property is not present on `Person`).
 
 The assumption that we can select/project parts of a subtype that make up a supertype is essential for keeping strictness in JSON Schemas.
 
-And unfortunately, specifying `{ "unevaluatedProperties": false }` does not behave as expected when composing types together in JSON Schema either, which means we will have to redefine how `{ "additionalProperties": true }` or `{ "unevaluatedProperties": false }` behaves within our type extension system to make supertypes keep strictness while allowing composition.
+We propose slight modifications to how `{ "additionalProperties": true }` and `{ "unevaluatedProperties": false }` behave and may be used within our type extension system to make supertypes keep strictness while allowing composition.
+
+Concrete examples of how JSON Schema breaks with these validation constraints are shown in the [Reference-level explanation](#problems-with-unevaluatedproperties)
+
+## Defining extended entity types
+
+Extended types will be defined with conventional JSON Schema syntax, the `allOf` keyword. An entity type can extend another entity type by adding an entry to `allOf` value with a versioned URI reference.
+Using a [versioned URI](https://github.com/blockprotocol/blockprotocol/blob/main/rfcs/text/0408-versioning-types.md#type-uris) makes it so that subtypes aren't automatically updated when the supertype is.
+
+As extended types can extend other extended types, we must also make sure that there are no cycles within the type hierarchy, as it makes types difficult to resolve/reason about, and could lead to unpredictable behavior.
+
+## Defining entity type duplication
 
-Concrete examples of how JSON Schema breaks with these validation constraints are shown in the [Reference-level explanation](#additional-properties-on-types-1)
+When looking for public types, if extending an entity type is not sufficient for a given situation, an alternative is to duplicate the type, such that individual properties can be overridden. Duplicated types become new complete, standalone types.
 
-## Defining extended types
+Type duplication, unlike extended types, will not allow for direct substitution, as the types may be incompatible, thus requiring some sort of mapping (which is yet to be defined within the type system).
```

Comment:
As the details of mapping are still fuzzy, I'm okay with making these changes. The intention I had about the "direct substitution" is that no mapping would be required because of assumptions made on type shape. As soon as we duplicate a type, we can't make that assumption, and we require mapping even if the substitution is direct and one-to-one.